ilcsoft/util.py

```diff
@@ -21,6 +21,13 @@
 import fnmatch
 import platform
 
+#--------------------------------------------------------------------------------
+def os_system( cmd ):
```

Comment:
Not sure how other systems would react if they don't use bash. Maybe a specific case for Ubuntu in this case?

Reply:
As we are explicitly sourcing a bash script for ROOT (thisroot.sh), there is not much point in allowing other shells than bash. For this a lot more work would be needed.